packages/monitoring/src/types/index.ts

```diff
@@ -0,0 +1,28 @@
+interface IServiceInfo {
+  ServiceName: string;
+}
+
+export interface IDisconnectHandler {
+  disconnect: () => Promise<void>;
+}
+export interface IServiceAliveness extends IServiceInfo {
+  isAlive: () => Promise<ServiceStatus>;
+}
+
+export type ServiceStatus = {
+  alive: boolean;
+  error?: Error;
+};
+
+export enum TextColor {
+  Reset = "\x1b[0m",
+  Black = "\x1b[30m",
+  Red = "\x1b[31m",
+  Green = "\x1b[32m",
+  Yellow = "\x1b[33m",
+  Blue = "\x1b[34m",
+  Magenta = "\x1b[35m",
+  Cyan = "\x1b[36m",
+  White = "\x1b[37m",
+  Gray = "\x1b[90m",
+}
```

Comment:
Checkout https://www.npmjs.com/package/chalk
Chalk is doing the same functionality without any configuration

Reply:
awesome , thanks